examples/test-dashboard/simple-agent.html

```diff
@@ -112,6 +112,7 @@ <h2>Read Records</h2>
 
   <script type="module">
     import { Web5 } from './browser.mjs';
+    import { DataStream } from 'https://cdn.jsdelivr.net/npm/@tbd54566975/dwn-sdk-js@0.0.26';
```

Comment:
@frankhinek @mistermoe this will be an extra load from the CDN, could that datastream be bundled into web5 js?

Reply:
we also could just load from the JS in `dist` instead

Reply:
no I am thinking more when loading web5 from the CDN (not for this specific example, as loading from dist won't help?)

Reply:
@michaelneale i believe it already is, i think it's `web5.dwn.sdk.DataStream` but i could be wrong

Reply:
This was purely for testing and can be ignored.  I need to do some work to update `simple-agent`